cookbooks/aws-parallelcluster-common/test/common/libraries/node.rb

```diff
@@ -0,0 +1,28 @@
+class ::Hash
+  def deep_merge(second)
+    merger = proc { |_key, v1, v2| v1.is_a?(Hash) && v2.is_a?(Hash) ? v1.merge(v2, &merger) : v2 }
+    merge(second, &merger)
+  end
+end
+
+class Node < Inspec.resource(1)
+  name 'node'
+
+  desc '
+    Node properties
+  '
+  example "
+    node.node['cluster']['cluster_admin_user']
```

Comment:
Nitpick: should not be `node['cluster']['cluster_admin_user']` since the name is node?